invokeai/backend/model_management/convert_ckpt_to_diffusers.py

```diff
@@ -1070,7 +1070,7 @@ def convert_controlnet_checkpoint(
     extract_ema,
     use_linear_projection=None,
     cross_attention_dim=None,
-    precision: torch.dtype = torch.float32,
+    precision: torch.dtype = None,
```

Comment:
Just suggestion, but to be precise with type annotation - as it now nullable, it should be defined as `Optional[torch.dtype]`.
And same for all other precision arg definitions.

Reply:
Thanks. Fixed.